modules/lvms-about-lvm-storage-installation.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
```

Comment:
```suggestion
// * storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
```

Reply:
Add a * per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata

Reply:
Updated as per the suggestion.